xbmc/pvr/PVRManager.cpp

```diff
@@ -150,7 +150,13 @@ void CPVRManager::OnSettingChanged(const CSetting *setting)
     return;
 
   const std::string &settingId = setting->GetId();
-  if (settingId == "pvrmanager.enabled")
+  if (settingId == "pvrmanager.backendchannelorder" || settingId == "pvrmanager.usebackendchannelnumbers")
+  {
+    // delegate call to the underlying groups
+    g_PVRChannelGroups->Get(true)->OnSettingChanged(setting);
```

Comment:
why don't you register the callback directly in `CPVRChannelGroup` instead of pvrmanager -> groups -> group?